tests/sentry/tsdb/test_snuba.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import pytz
+from datetime import datetime, timedelta
+
+from sentry.testutils.cases import OutcomesSnubaTest
+from sentry.tsdb.base import TSDBModel
+from sentry.tsdb.snuba import SnubaTSDB
+from sentry.utils.dates import to_timestamp
+from sentry.utils.outcomes import Outcome
+
+
+def floor_to_hour_epoch(value):
+    value = value.replace(minute=0, second=0, microsecond=0)
+    return int(to_timestamp(value))
+
+
+class SnubaTSDBTest(OutcomesSnubaTest):
+    def setUp(self):
+        super(SnubaTSDBTest, self).setUp()
+        self.db = SnubaTSDB()
+
+        # Set up the times
+        self.now = datetime.now(pytz.utc)
+        self.start_time = self.now - timedelta(days=7)
+        self.one_day_later = self.start_time + timedelta(days=1)
+
+    def test_organization_outcomes(self):
+        for tsdb_model, outcome in [
+            (TSDBModel.organization_total_received, Outcome.ACCEPTED),
+            (TSDBModel.organization_total_rejected, Outcome.RATE_LIMITED),
+            (TSDBModel.organization_total_blacklisted, Outcome.FILTERED),
+        ]:
+            # Create all the outcomes we will be querying
+            self.store_outcomes(
+                self.organization.id, self.project.id, outcome.value, self.start_time, 3
+            )
+            self.store_outcomes(
+                self.organization.id, self.project.id, outcome.value, self.one_day_later, 4
+            )
```

Comment:
Might not be a bad idea to create some outcomes that you're *not* expecting to show up in the result set (out of time range, different organization) to ensure query conditions are being applied accurately.